Wormholy.xcodeproj/project.pbxproj

```diff
@@ -290,9 +291,9 @@
 			isa = PBXGroup;
 			children = (
 				8933C7811EB5B7E0000D00A4 /* Sources */,
-				8933C7831EB5B7EB000D00A4 /* Tests */,
 				52D6D99C1BEFF38C002C0205 /* Configs */,
 				4508718A207E59FF00EE2D31 /* WormholyDemo */,
+				E81B8F4F2338DA8D005E381F /* WormholyTests */,
```

Comment:
Xcode was failing while attempting to load the underlying module for the tests. Deleting the old test target and making a new one fixed it